src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.html

```diff
@@ -22,7 +23,11 @@
         'margin-top': activeScenarios.length + archivedScenarios.length === 0
       }"
       emptyStateTitle="No scenarios yet"
-      emptyStateContent="Click “New scenario” to start creating your scenarios.">
+      [emptyStateContent]="
+        canAddScenarioForPlan()
+          ? 'Click “New scenario” to start creating your scenarios.'
+          : 'No scenarios exist yet for this planning area.'
```

Comment:
I'm asking for confirmation on this wording.

Reply:
good catch on this!